### DISCUSSION TITLE
Need documentation/presentation on GA

### DISCUSSION THREAD

**Discussion by ksbhuvanesh** [community/community#139669] [Actions] [answered]
### Select Topic Area

General

### Body

Hi There,
I am looking for a documentation/presentation on GA which covers all the features for CI/CD.

**Comment by fruetalo182** [comment, marked as answer]
Hi @ksbhuvanesh 
Here is some documentation:
[Oficial Github Actions Docs](https://docs.github.com/en/actions/about-github-actions/understanding-github-actions)
[Introduction to CI-CD and Actions](https://medium.com/@edoter92/github-actions-ci-cd-workflow-b0e970bdacad)
